src/main/java/net/daporkchop/fp2/asm/core/server/IMixinWorldBorder.java

```diff
@@ -0,0 +1,14 @@
+package net.daporkchop.fp2.asm.core.server;
+
+import net.minecraft.world.border.WorldBorder;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(WorldBorder.class)
+public interface IMixinWorldBorder {
+    @Accessor("startDiameter")
+    double getWorldBorderStartDiameter();
+
+    @Accessor("worldSize")
+    int getWorldSize();
```

Comment:
why? `WorldBorder` already has a public method `getSize()` which returns `worldSize`. there is no reason to use a mixin accessor _or_ an access transformer for this at all.